packages/neuron-ui/src/contexts/WalletNames.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react'
+
+export interface WalletNames {
```

Comment:
As pointed out above, we should have access to `wallets` list, not names from UI.